src/App.svelte

```diff
@@ -51,8 +52,19 @@
     };
   }
 
+  function setQueryString(ctx, next) {
```

Comment:
I'm pretty sure we should be able to avoid managing a global queryString object by just setting it on the context via a handler, then handling it in the `setComponent` method. This has the added bonus of making it possible for us to use the query string in other pages in the future (something we will almost certainly want to do).

Search for `query-string` in https://visionmedia.github.io/page.js/ for some code examples which should help you out here.